p/scripts/main.js

```diff
@@ -426,10 +426,17 @@ function toggleContent(new_active, old_active, skipping) {
 
 	if (context.sticky_post) {	// Stick the article to the top when opened
 		const prev_article = new_active.previousElementSibling;
-		let new_pos = new_active.offsetParent.offsetTop + new_active.offsetTop;
+		const nav_menu = document.querySelector('.nav_menu');
+		let nav_menu_height = 0;
+
+		if (getComputedStyle(nav_menu).position === 'fixed' || getComputedStyle(nav_menu).position === 'sticky') {
```

Comment:
I have not tested yet. Any impact on the other themes?

Reply:
I tested it with Origine. So it should not have an negative impact on other themes.

Let me explain this lines:
The changes in the JS file are needed when the user navigates with the left, top, right arrow navigation through the feed list. An additional factor was needed to calculate the correct position of the opened article.